tests/mock_client.php

```diff
@@ -6,14 +6,15 @@
 use Recurly\Utils;
 use PHPUnit\Framework\MockObject\Generator;
 use Recurly\HttpAdapter;
+use Psr\Log\LoggerInterface;
 
 class MockClient extends BaseClient
 {
     use Recurly\RecurlyTraits;
 
-    public function __construct()
+    public function __construct($logger)
     {
-        parent::__construct("apikey");
+        parent::__construct("apikey", $logger);
```

Comment:
I wonder if it would be good to create the logger here and then remove the pass-through of a logger from the MockClient callers, which don't currently need to send the `"apikey"` mock. Unless there's a consequence to this I'm not seeing?

Reply:
I updated the mock client to create a default logger, but it needs to accept a logger for this test: https://github.com/recurly/recurly-client-php/pull/506/files#diff-0491b55fddb5c4dff4c63cc5b0fa7a19R30